substrate/client/consensus/common/Cargo.toml

```diff
@@ -20,7 +20,7 @@ async-trait = "0.1.74"
 futures = { version = "0.3.21", features = ["thread-pool"] }
 futures-timer = "3.0.1"
 libp2p-identity = { version = "0.1.3", features = ["ed25519", "peerid"] }
-log = "0.4.17"
+log = { workspace = true }
```

Comment:
In the client crates we should enable `std` by default?

Reply:
Ah yes, i thought that just writing this would enable them again, but it does not.  
Changed it now.